### PR TITLE
Add ChallengeShopper and IdentifyShopper PaymentsResponse 3DS2 checkout

### PR DESCRIPTION
**Pull request by AlexandrosMor**
Add ChallengeShopper and IdentifyShopper PaymentsResponse result code for 3DS2 checkout

### CODE REVIEW

File: Adyen.EcommLibrary.Test/CheckoutTest.cs [hunk 1]
```diff
@@ -41,6 +41,21 @@ public void PaymentsTest()
...
         }
 
+        /// <summary>
+        /// Test success flow for 3DS2
+        /// POST /payments
+        /// </summary>
+        [TestMethod]
+        public void Payments3DS2Test()
+        {
+            var payment3DS2Request = CreatePaymentRequest3D2S();
```
Comment (Aleffio) [inline Adyen.EcommLibrary.Test/CheckoutTest.cs]: minor
```suggestion
            var payment3DS2Request = CreatePaymentRequest3DS2();
```

File: Adyen.EcommLibrary.Test/BaseTest.cs [hunk 2]
```diff
@@ -118,6 +118,26 @@ public Model.Checkout.PaymentRequest CreatePaymentRequestCheckout()
             return paymentsRequest;
         }
 
+        /// <summary>
+        /// 3DS2 payemnts request
+        /// </summary>
+        /// <returns></returns>
+        public Model.Checkout.PaymentRequest CreatePaymentRequest3D2S()
```
Comment (Aleffio) [inline Adyen.EcommLibrary.Test/BaseTest.cs]: minor
```suggestion
        public Model.Checkout.PaymentRequest CreatePaymentRequest3DS2()
```

File: Adyen.EcommLibrary.Test/CheckoutTest.cs (hunk 1 above)
Comment (AlexandrosMor) [reply Adyen.EcommLibrary.Test/CheckoutTest.cs]: Done 👍

File: Adyen.EcommLibrary.Test/BaseTest.cs [hunk 3]
```diff
@@ -118,6 +118,26 @@ public Model.Checkout.PaymentRequest CreatePaymentRequestCheckout()
             return paymentsRequest;
         }
 
+        /// <summary>
+        /// 3DS2 payemnts request
```
Comment (rikterbeek) [inline Adyen.EcommLibrary.Test/BaseTest.cs]: type payemnts => payments

**Comment by AlexandrosMor** [comment]
Fixes #75

**Pull request merged**